packages/back-end/src/routers/event-webhooks/event-webhooks.controller.ts

```diff
@@ -33,8 +34,6 @@ export const getEventWebHooks = async (
   req: GetEventWebHooksRequest,
   res: Response<GetEventWebHooks>
 ) => {
-  req.checkPermissions("manageWebhooks");
```

Comment:
That isn't quite true.  We do the same for secret API keys where only admins can view them.  Event Webhooks are similar since they contain a signing secret that we don't want to expose to all users in the org.

Let's create `canView` helper functions for webhooks that can just check for `manageWebhooks` under the hood for now.